en/includes/references/grant-types.md

```diff
@@ -52,29 +114,87 @@ The following diagram shows how the refresh token flow works.
+    
+    === "Sample request (/token)"
+
+        ```bash
+        curl -k {{base_url_example}}/oauth2/token \
+        --header "Content-Type: application/x-www-form-urlencoded"
+        --header "Authorization: Basic RWkwV2Y5YnpmTXE0UTBsZndTdlRQamU4a2NFYTpIRvvyUzJIUjlrZE9YMjBXTG9JNmY1eE1wdUlBamdKeG5aUVVUMV9lNTJnYQ==" \
+        --data-urlencode "grant_type=refresh_token" \
+        --data-urlencode "refresh_token=96a6dsd97-0120-3bec-86be-21b58f600a07"
+        ```
+
 4. {{ product_name }} sends a new access token and a new refresh token.
 
+    ```json
+    {
+    "access_token":"b9ed0658-f187-3d39-a4f1-6d42522e1eee",
+    "refresh_token":"3426ff78-62a5-32fa-be6e-74ab69d4cbf4",
+    "token_type":"Bearer",
+    "expires_in":3600
+    }
+    ```
 
 ## Client credentials grant
 
-The client credentials flow provides a secure way for client applications to obtain an access token without user authentication. This can be useful in scenarios where the client application needs to access its own resources, such as data storage or APIs, but does not require access to user data.
+The client credentials flow provides a secure way for client applications to obtain an access token without user authentication. This can be useful in scenarios where the client application needs to access its own resources, such as data storage or APIs, without requiring user authentication.
 However, it is important to ensure that the client credentials are kept secure, as any party that posses these credentials can obtain access tokens and access the client's resources.
 
 The following diagram shows how the client credentials grant flow works.
 
 ![How the client credentials grant works]({{base_path}}/assets/img/references/grants/client-credentials.png)
 
 1. The client application sends its credentials (`client_id` and `client_secret`) to {{ product_name }} and requests an access token.
+
+    === "Request format (/token)"
+
+        ```bash
+        curl -v -k -X POST {{base_url}}/oauth/token \
+        --header "Authorization: Basic <Base64Encoded(CLIENT_ID:CLIENT_SECRET)>" \
+        --header "Content-Type:application/x-www-form-urlencoded" \
+        --data-urlencode "grant_type=client_credentials" 
+        ```
+    
+    === "Sample request (/token)"
+
+        ```bash
+        curl -v -k -X POST {{base_url_example}}/oauth/token \
+        --header "Authorization: Basic RWkwV2Y5YnpmTXE0UTBsZndTdlRQamU4a2NFYTpIRvvyUzJIUjlrZE9YMjBXTG9JNmY1eE1wdUlBamdKeG5aUVVUMV9lNTJnYQ==" \
+        --header "Content-Type:application/x-www-form-urlencoded" \
+        --data-urlencode "grant_type=client_credentials"
+        ```
+
 2. {{ product_name }} sends the access token to the client application.
-3. The client application can now request user information from the resource server by providing the access token.
-4. The resource server returns the requested user information to the client application.
+
+    ```json
+    {
+    "token_type":"Bearer",
+    "expires_in":2061,
```

Comment:
Shall we stick adding 3600 as the expiry time , other wise the readers may confuse the default expiry times are different in different grant types.